data_reports/999_References.Rmd

```diff
@@ -3,13 +3,14 @@
 ## ISCN3 citations
 
 data_report number | dataset_name | citation
--------------|----------
+
```

Comment:
add back in `------|------|-------` here to add a horizontal line in the markdown table.